packages/sui-ssr/status-codes.js

```diff
@@ -0,0 +1,11 @@
+export const statusCodes = {
```

Comment:
This object can grow as we want in further iterations.

Reply:
`307` and `308` look like interesting options, too

`307`: _The Web page is temporarily unavailable for unforeseen reasons._
`308`: _Reorganization of a Web site, with non-GET links/operations._

https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections#temporary_redirections